LEAF_Nexus/sources/Employee.php

```diff
@@ -441,6 +441,83 @@ public function lookupFirstName($firstName)
         return $result;
     }
 
+    public function lookupAllUsersLastName($lastName)
+    {
+        $lastName = $this->parseWildcard($lastName);
+
+        $sql = "SELECT * FROM {$this->tableName}
+                    WHERE lastName LIKE :lastName
+                    AND deleted >= 0
+                    ORDER BY {$this->sortBy} {$this->sortDir}
+                    {$this->limit}";
+
+        $vars = array(':lastName' => $lastName);
+        $result = $this->db->prepared_query($sql, $vars);
+
+        if (count($result) == 0)
+        {
```

Comment:
Take a look at PSR12 - 5.# for if, switch, for, etc on how to layout the structure.

the { needs to be on the same line as the if unless you have multiple conditions and you separate those onto separate lines. Same goes for loops and switches.

Reply:
Will do, thank you!